### ISSUE TITLE
Unreadable text on gnome Software

### ISSUE THREAD

**Issue by bil-elmoussaoui** [tista500/Adapta#37]
As you can see in this screenshot! the tabs are unreadable 
![capture du 2016-04-05 14-58-22](https://cloud.githubusercontent.com/assets/7660997/14282500/f193769c-fb3e-11e5-920f-156990be68d0.png)

**Comment by tista500** [comment]
@bil-elmoussaoui,

This is the duplicate of this issue: #36 
And already fixed in master...

Regards.

**Comment by bil-elmoussaoui** [comment]
Well, i'm using the latest version from master and still not fixed, i'm using Arch with Gtk 3.20

**Comment by tista500** [comment]
@bil-elmoussaoui,

:(
So which version of Gnome-Software you run? Mine is 3.20.1 on Ubuntu GNOME 16.04 & Gtk+ 3.20.2 and seems fixed:
![screenshot from 2016-04-05 23-28-11](https://cloud.githubusercontent.com/assets/6073622/14285387/3b3a0c38-fb86-11e5-842d-238416a29de3.png)

New could you please attach your current `/usr/share/themes/Adapta/gtk-3.20/gtk.css`? (Do not put it into inline texts since file will be too huge, so please attach as txt file.)

Regards.

**Comment by bil-elmoussaoui** [comment]
I'm using Gnome Software 3.20.0 (it's might be an issue with that version of Gnome Software?)


[gtk.txt](https://github.com/tista500/Adapta/files/204832/gtk.txt)

**Comment by tista500** [comment]
@bil-elmoussaoui,

Just now I've finished bisecting, and fixes are included already in yours.
Then I've also compiled 3.20.0 from Gnome's git: https://git.gnome.org/browse/gnome-software/ without any specific patches. Finally still fixes are valid even if the version 3.20.0 are installed:
![screenshot from 2016-04-06 01-36-36](https://cloud.githubusercontent.com/assets/6073622/14290112/b6fd681c-fb98-11e5-9c3b-c1d3b7a31baa.png)

So I can't reproduce your (Arch?) issue anyway...

Regards.
